### PR TITLE
test: fix generate-sym-test using the wrong array

### PR DESCRIPTION
**Pull request by fedegiova**
For my understanding bsearch is searching in the wrong array. Or, if it's the right one, then the size is wrong. In another commit I made the arrays different by mistake and that triggered a SIGSEV during tests.

### CODE REVIEW

File: src/test/generate-sym-test.py [hunk 1]
```diff
@@ -105,9 +105,9 @@ def process_source_file(file):
         }
 
         for (j = 0; symbols_from_source[j].name; j++) {
-                struct symbol*n = bsearch(symbols_from_source+j, symbols_from_source, sizeof(symbols_from_sym)/sizeof(symbols_from_sym[0])-1, sizeof(symbols_from_sym[0]), sort_callback);
```
Comment (keszybz) [inline src/test/generate-sym-test.py:108]: Please add a second commit which changes each `symbol*n` to `symbol *n`.

Comment (fedegiova) [reply src/test/generate-sym-test.py:108]: Done. Do you have a code format tool?

**Comment by keszybz** [comment]
The tests passed previously, except for some unrelated failures. No need to merge for the CI for the formatting fix.

File: src/test/generate-sym-test.py (hunk 1 above)
Comment (keszybz) [reply src/test/generate-sym-test.py:108]: People use various formatters. But this is a generator of code, so most formatters wouldn't know how to deal with it.

**Pull request merged**